yt/utilities/tests/test_set_log_level.py

```diff
@@ -2,6 +2,19 @@
 
 from yt.utilities.logger import set_log_level
 
+old_level = None
```

Comment:
is this line actually needed ? I feel like it might not be (though maybe a `NameError` is more confusing than an `AttributeError` ?)

Reply:
It works fine without, but pylint complains about it

Reply:
[personal opinion] I don't really care for pylint, especially in yt where we don't use it. [/personal opinion]
Anyway I didn't mean to rant, this is fine as is :)